core/citrus-api/src/main/java/com/consol/citrus/util/TypeConverter.java

```diff
@@ -21,6 +21,11 @@ public interface TypeConverter {
 
     Map<String, TypeConverter> converters = new HashMap<>();
 
+    String DEFAULT = "default";
+    String SPRING = "spring";
+    String APACHE_CAMEL = "camel";
+    String GROOVY = "groovy";
```

Comment:
Users should have the opportunity to add new type converter implementations in their projects. so the code should be using String as a generic identifier for type converters instead of the fixed enum values.

Reply:
ah, I see. thx for the explanation.